### ISSUE TITLE
ScriptReader.readStatement can return string which only includes whitespace

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#347]
```
This is not a problem for H2, but I use RunScript.execute against both H2 and 
MySQL and the latter rejects the empty string complaining about an empty query. 
 Fixed by returning null when the trimmed string is empty.
```

Original issue reported on code.google.com by `g...@maginatics.com` on 22 Sep 2011 at 11:49

**Comment by GoogleCodeExporter** [comment]
```
Sounds like a good idea. I will have a look.
```

Original comment by `thomas.t...@gmail.com` on 10 Oct 2011 at 4:55

**Comment by GoogleCodeExporter** [comment]
```
Hi,

This doesn't work as expected: it will return null when reading an empty 
statement. However, null is the indication that the end of the file was reached.

Instead, the RunScript and Shell tool should ignore empty statements. I will 
change that.

Regards,
Thomas
```

Original comment by `thomas.t...@gmail.com` on 18 Oct 2011 at 6:49
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.3.161
```

Original comment by `thomas.t...@gmail.com` on 28 Oct 2011 at 3:56
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Verified fix.  Thanks!
```

Original comment by `g...@maginatics.com` on 29 Oct 2011 at 8:25
